recipes/openfaas.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: kbrew
+app:
+  args:
+    functionNamespace: openfaas-fn
+    generateBasicAuth: true
+  repository:
+    name: openfaas
+    url: https://openfaas.github.io/faas-netes/
+    type: helm
+  namespace: "openfaas"
+  sha256:
+  version: 8.0.2
+  pre_install:
+  - steps:
+    - kubectl create ns openfaas-fn
```

Comment:
@dirien 
The namespace creation should be done via kubectl apply https://.... and using the namespaces from the faas-netes repo

Reply:
done